### ISSUE TITLE
Crash when qimgv try to show a text file with .svg ext

### ISSUE THREAD

**Issue by linsui** [easymodo/qimgv#132]
It's ease to make such a file. Just write some letters in a text file and save as a svg file.
e.g.

`abcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcd`

**Comment by linsui** [comment]
Fixed in dev build.

**Comment by linsui** [comment]
This problem still exist in 0.8.8 .

**Comment by easymodo** [comment]
are you sure you opened it in 0.8.8? doesn't crash for me

upload that file

**Comment by linsui** [comment]
Yes, it's 0.8.8 . I just save above string `abcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcdeabcd` in a text file as `a.svg` and put it in a directory with some other picture. Then I open a picture and switch to the svg file. qimgv crashes.
